templates/docs/examples/patterns/navigation/_full-width-dropdown.jinja

```diff
@@ -0,0 +1,87 @@
+{% macro dropdown(id, title) %}
+<div class="p-navigation__dropdown is-full-width is-collapsed"  data-level="1" id="{{ id }}" aria-hidden="true">
+  <div class="p-navigation__dropdown-content-mobile">
+    <ul class="p-list u-no-margin">{# TODO: use p-list temporarly, we will later have more specific styling class name for it #}
+      <li class="p-navigation__item--dropdown-close" id="{{ id }}-back">
+        <button aria-controls="{{ id }}" class="p-navigation__link js-back">
+          Back
+        </button>
+      </li>
+      <li>
+        <a href="#" class="p-navigation__dropdown-item">TODO...</a>
+      </li>
+      <li>
+        <a href="#" class="p-navigation__dropdown-item">TODO....</a>
+      </li>
+    </ul>
+  </div>
+  <div class="p-navigation__dropdown-content">
```

Comment:
Fair point. I assumed it would be a "default" (desktop content), and mobile as an alternative.

Also "small" "large" could be confusing here, as navigation does not use our standard breakpoints, but has mobile/desktop view based on amount of elements in the nav, so is switching between 2 views at a breakpoint that is configurable, and may not be aligned with rest of Vanilla.

Reply:
Renamed to `p-navigation__dropdown-content--sliding` and `p-navigation__dropdown-content--full-width` for consistency and clarity.